components/widgets/ui/src/main/resources/PhenoTips/Widgets.xml

```diff
@@ -2397,9 +2397,8 @@ return PhenoTips;
         onSuccess: function (response) {
           this.setContent(this.buildTree(this._getDataFromResponse(response)));
           this.__crtRoot = id;
-          if (this.container.content) {
-            Event.fire(document, 'xwiki:dom:updated', {'elements' : [this.container.content]});
-          }
+          if (this.container.contentContainer != null)
```

Comment:
Not just this change in particular, but the fact that sometimes `content` is needed, other times `contentContainer` is used.